vjunosevolved/docker/launch.py

```diff
@@ -55,6 +54,7 @@ def __init__(self, hostname, username, password, conn_mode):
             driveif="virtio",
             cpu="IvyBridge,vme=on,ss=on,vmx=on,f16c=on,rdrand=on,hypervisor=on,arat=on,tsc-adjust=on,umip=on,arch-capabilities=on,pdpe1gb=on,skip-l1dfl-vmentry=on,pschange-mc-no=on,bmi1=off,avx2=off,bmi2=off,erms=off,invpcid=off,rdseed=off,adx=off,smap=off,xsaveopt=off,abm=off,svm=off",
             smp="4,sockets=1,cores=4,threads=1",
+            mgmt_passthrough=False,
```

Comment:
@vista- I flipped the default for vjunos* platforms to host-forward mode. I want to let people try this new mode first for a couple of releases, and then make the switch for all platforms.

Ideally we should get some bug reports when people try it out in a controlled env

Reply:
Sounds good!